lib/timeline/locales.js

```diff
@@ -29,15 +31,17 @@ exports['nl_BE'] = exports['nl'];
 require('moment/locale/de');
 exports['de'] = {
   current: 'Aktuelle',
-  time: 'Zeit'
+  time: 'Zeit',
+  deleteSelected: 'L\u00f6sche Auswahl',
 };
 exports['de_DE'] = exports['de'];
 
 // French
 require('moment/locale/fr');
 exports['fr'] = {
   current: 'actuel',
-  time: 'heure'
+  time: 'heure',
+  deleteSelected: 'Effacer la selection',
```

Comment:
`supprimer l'élément sélectionné`

Reply:
This is taken from https://github.com/visjs/vis-network/blob/505ef4e9aa5c500a7597523c0eed7f0ad27d45c0/lib/network/locales.js#L170
If this is wrong, vis-network is also wrong